examples/nodejs/src/marlowe-object-flow.ts

```diff
@@ -295,17 +310,23 @@ async function contractMenu(
   });
   switch (action.actionType) {
     case "check-state":
-      return contractMenu(lifecycle, scheme, contractId);
+      return contractMenu(lifecycle, scheme, sourceMap, contractId);
     case "advance":
     case "deposit":
       if (!action.results) throw new Error("This should not happen");
       console.log("Applying input");
       const txId = await lifecycle.contracts.applyInputs(contractId, {
         inputs: action.results.inputs,
+        invalidBefore: posixTimeToIso8601(
+          action.results.environment.timeInterval.from
+        ),
+        invalidHereafter: posixTimeToIso8601(
+          action.results.environment.timeInterval.to
+        ),
```

Comment:
I want to modify getApplicable inputs to offer an `applyInputs` that fill the inputs and env parameter and the user can add other properties like tags